docs/UsersGuide/source/ConfigNewPlatform.rst

```diff
@@ -372,6 +370,9 @@ Those requirements highlighted in **bold** are included in the NCEPLIBS-external
    * **libPNG**
    * **zlib**
 
+..
+   COMMENT: Update version of ESMF? Need other version updates?
```

Comment:
Does this need to be addressed?

Reply:
I updated the version of ESMF, but I think software versions/requirements are still in flux, and I will have to make final updates closer to the release.